runner/steps.ts

```diff
@@ -161,7 +161,9 @@ function runBrowsers(context: Context) {
       // TODO(nevir): Better rationalize run-end and hook.
       await context.emitHook('cleanup');
 
-      return error;
+      if (error) {
+        throw error;
```

Comment:
(Weird, github wasn't showing me this conversation.)

Ideally we should follow the robustness principle wrt errors. _We_ should only throw Error objects, and we should try to handle any objects that are thrown. The problem is that it's pretty damn tedious to handle non-Error errors. Want to check for `.message` or `.stack` every time before using them? Ugh. Rethrowing is another ugly case. We don't want to wrap every rethrow.

So generally I just assume Error objects until proven otherwise. If we have plugins throwing strings, then we can deal.